x-pack/elastic-agent/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -323,6 +323,24 @@ func (c *enrollCmd) prepareFleetTLS() error {
 	return nil
 }
 
+func (c *enrollCmd) daemonReloadWithBackoff(ctx context.Context) error {
+	err := c.daemonReload(ctx)
+	signal := make(chan struct{})
+	backExp := backoff.NewExpBackoff(signal, 60*time.Second, 10*time.Minute)
```

Comment:
Can we start with lower amounts? This seems rather large initial wait. Maybe lets do like 10 seconds to maximum of 1 minute?

This is a local daemon it should be quick to connect and communicate.